### ISSUE TITLE
Content Picker Field's Contained Content Type -> "Select all" should include any content items added later

### ISSUE THREAD

**Issue by stantoxt** [OrchardCMS/OrchardCore#4700]
If we want to pick a content item of a new content type which hasn't been added to the contentpickerpart, we have to go to pick content type for the contentpickerpart, and this operation may be a little inconvenient.I hope contentpickerpart can automatically search all content items of all content types.
1.Can we add select all checkbox?
2.Can we add stereotypes input to select all content types which own these stereotypes ?

**Comment by sebastienros** [comment]
These are valid suggestions.

- All Content Types
- Specific Content Types
- Content types of a specific Stereotype

**Comment by stantoxt** [comment]
@sebastienros thanks.
At now we already have the Specific Content Types,so we need implementation of the other two.

**Comment by Piedone** [comment]
Same for Bag Part, Flow Part, and anything else that has a similar content type selector.

**Comment by hishamco** [comment]
If I'm not we already have all the options, @MikeAlhayek correct me if I'm wrong

**Comment by Piedone** [comment]
No, "Select all" just checks all the checkboxes currently visible. If a new content type is added later, then you have to go back to the part's editor, and check the corresponding checkbox too:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/ca74b047-a122-4c65-b863-c2f8a220f22b)

**Comment by hishamco** [comment]
Right, I will have a look

**Comment by Piedone** [comment]
This isn't something that I'd sweat about too much for now, we have much more important things to tackle.

**Comment by hishamco** [comment]
Sure, even I :)
